openlibrary/templates/work_search.html

```diff
@@ -147,16 +147,24 @@ <h1>$_("Search")</h1>
         $# results because of performance issues due to increased load. See
         $# this commit to revert.
         <center>
-            <span class="red">$_("No results found.")</span>
+          <div class="red">No results found.</div>
```

Comment:
We want these strings wrapped in `$_("...")`, that's what makes these strings translatable!

```suggestion
          <div class="red">$_("No results found.")</div>
```